### PR TITLE
feat: xud created geth keystore

### PR DESCRIPTION
**Pull request by reliveyy**
Resolves https://github.com/ExchangeUnion/xud-docker/issues/177

### CODE REVIEW

**Comment by reliveyy** [comment]
> Please push images! @reliveyy

Pushed geth, raiden and xud. Now you can test it.

File: images/raiden/entrypoint.sh [hunk 1]
```diff
@@ -1,30 +1,88 @@
...
+  "--datadir $RAIDEN_DIR"
+  "--api-address 0.0.0.0:5001"
+  "--matrix-server https://raidentransport.exchangeunion.com"
+  "--address $(get_addr)"
+  "--keystore-path $KEYSTORE_DIR"
+)
 
-# Address must be EIP55 checksummed
-addr=`python /checksum.py $addr`
+echo "Options are ${OPTS[*]}"
```
Comment (erkarl) [inline images/raiden/entrypoint.sh]: Let's delete this - I added it previously for debugging purposes.

File: xud-mainnet/docker-compose.yml [hunk 2]
```diff
@@ -94,28 +88,10 @@ services:
...
-      --routing-mode private
-      --environment-type production
-      --tokennetwork-registry-contract-address 0xd32F5E0fF172d41a20b32B6DAb17948B257aa371
-      --secret-registry-contract-address 0x322681a720690F174a4071DBEdB51D86E7B9FF84
-      --service-registry-contract-address 0x281937D366C7bCE202481c45d613F67500b93E69
-      --user-deposit-contract-address 0x4F26957E8fd331D53DD60feE77533FBE7564F5Fe
-      --monitoring-service-contract-address 0x37cC37D7703554183aE544391945e7D0588b7693
-      --one-to-n-contract-address 0x3dda5BE50Af796618d9f48c021Cf0C9FD64FFeb1
     volumes:
       - ./data/geth:/root/.ethereum
```
Comment (erkarl) [inline xud-mainnet/docker-compose.yml]: Is raiden still using geth's volume or can we remove it?

File: images/raiden/entrypoint.sh [hunk 3]
```diff
@@ -1,30 +1,88 @@
...
-fi
+source /opt/venv/bin/activate
 
-if [[ ! -e ~/.raiden/config.toml ]]; then
-    touch ~/.raiden/config.toml
-fi
+XUD_DIR="/root/.xud"
+case $NETWORK in
+testnet)
+  KEYSTORE_DIR="$XUD_DIR/keystore"
```
[the remark this answers is not in the document]
Comment (reliveyy) [reply images/raiden/entrypoint.sh]: It's not in the same environment. `testnet/data/xud/keystore` and `mainnet/data/xud/keystore`

[the remark this answers is not in the document]
Comment (erkarl) [reply images/raiden/entrypoint.sh]: Oh, right. Thanks for pointing it out.

**Pull request merged**